pyproject.toml

```diff
@@ -96,6 +89,9 @@ addopts = [
     "--strict-markers",
 ]
 
-
 [tool.pytest.ini_options.markers]
 no_django_db = "mark tests that should not be marked with django_db."
+
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
```

Comment:
qu'est-ce que c'est ?

Reply:
bien vu ! j'ai utilisé l'outil de migration `uvx migrate-to-uv` et je n'ai pas encore exploré cette partie du resultat. à creuser …

Reply:
[build-system](https://docs.astral.sh/uv/concepts/projects/config/#build-systems) 
Necessaire si besoin d'installer le projet comme un package dans l'environnement virtuel. Ce n'est pas le cas de la commu => à retirer.
Ça explique le problème autour du param `--no-editable` devenu inutile, et vraisemblablement la variable `UV_PROJECT_ENVIRONMENT` sera inutile également.

Reply:
le param `--no-emit-project` devient inutile également.